electrum/gui/qml/qetxfinalizer.py

```diff
@@ -210,14 +222,38 @@ def update_from_tx(self, tx):
         self.fee = QEAmount(amount_sat=int(fee))
         self.feeRate = f'{feerate:.1f}'
 
+        self.update_inputs_from_tx(tx)
         self.update_outputs_from_tx(tx)
```

Comment:
You need to do something similar in `QETxCpfpFeeBumper.update()`, near (new) line 868.
There only `update_outputs_from_tx` is called atm, so either call either both (`update_inputs_from_tx` and `update_outputs_from_tx`),
or maybe call `update_from_tx` instead, though the other side-effects of that are not relevant.

Reply:
thx, good catch